config/config.exs

```diff
@@ -0,0 +1,4 @@
+use Mix.Config
+
+config :ex_unit,
+  assert_receive_timeout: 1000
```

Comment:
And I am suggest to use 200 ms delay for start/stop to eliminate this kind of race condition:
```
   1) Rollbax.ClientTest: failure on setup_all callback, tests invalidated
     ** (MatchError) no match of right hand side value: {:error, {:already_started, #PID<0.508.0>}}
     stacktrace:
       test/rollbax/client_test.exs:7: Rollbax.ClientTest.__ex_unit_setup_all_1/1
       test/rollbax/client_test.exs:1: Rollbax.ClientTest.__ex_unit__/2
```